client/lib/options.js

```diff
@@ -23,7 +23,7 @@ define([], function () {
    * if a required option is missing
    */
   function checkRequired(requiredOptions, options) {
-    for (var i = 0, requiredOption; requiredOption = requiredOptions[i]; ++i) {
+    for (var i = 0, requiredOption; requiredOption = requiredOptions[i]; ++i) { //eslint-disable-line no-cond-assign
```

Comment:
It looks like we need to assign here. So i disabled lint, is that right? @pdehaan